packages/styles/src/components/text-input/obfuscatedField.js

```diff
@@ -10,9 +14,11 @@ class ObfuscatedField {
     btn.addEventListener("click", (e) => {
```

Comment:
The `aria-label` needs to be toggled as well from "show password" to "hide password". Can this be used for non-password fields like SSN? Maybe we should use more generic terminology like "show entry" and "hide entry"?

Reply:
done.